pkg/printer/table.go

```diff
@@ -21,10 +21,24 @@ func PrintTable(out io.Writer, v interface{}, getRow func(row interface{}) []int
 
 	table := NewTableWriter(out)
 	if len(headers) > 0 {
-		fmt.Fprintln(table, headers...)
+		fmt.Fprintln(table, tabify(headers)...)
 	}
 	for i := 0; i < rows.Len(); i++ {
-		fmt.Fprintln(table, getRow(rows.Index(i).Interface())...)
+		fmt.Fprintln(table, tabify(getRow(rows.Index(i).Interface()))...)
 	}
 	return table.Flush()
 }
+
+// tabify is a helper function which takes a slice and injects tab characters
+// between each element such that tabwriter can work its magic
+func tabify(untabified []interface{}) []interface{} {
```

Comment:
D'oh! I always use a better package than the stdlib `text/tabwriter` and completely forgot to do this.

I think I'll do a follow-up later to switch to a table writer package that I like better but thank you, this fixes the stdlib for now. 👍